mod-distribution/pom.xml

```diff
@@ -8,11 +8,11 @@
     <parent>
         <groupId>at.forsyte.apalache</groupId>
         <artifactId>apalache</artifactId>
-        <version>0.8.1-SNAPSHOT</version>
+        <version>0.8.1</version>
```

Comment:
@konnov Note that I've also configured the release process to simply version releases as the plain version number, without the `-RELEASE` suffix. This follows other mature projects I'm aware of, wherein you know a version is a principle release through the absence of any qualifiers or suffixes. Just want to be sure you're OK with this :)

Reply:
Yes. That sounds great. But maybe we should check with maven central or similar repos. It may be a maven thing to add `RELEASE`.

Reply:
Here are a couple examples of release versions:

- https://github.com/google/guice/blob/4.2.3/pom.xml#L30
- https://github.com/scoverage/scoverage-maven-plugin/blob/scoverage-maven-plugin-1.4.1/pom.xml#L31

Here's an explainer on Maven version conventions, seeming to indicate that plain `major.minor.patch` versions are standard (but also explaining when we'd want to use qualifiers).

Seems we are safe to drop it? Unless you prefer otherwise.